ios/brave-ios/Sources/Brave/Frontend/Sync/SyncAddDevice/SyncAddDeviceDetailView.swift

```diff
@@ -0,0 +1,483 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import BraveShared
+import Shared
+import UIKit
+
+class SyncAddDeviceInformationView: UIStackView {
```

Comment:
Why inherit UIStackView? Usually you inherit UIView, then add a stack-view as a child and insert into that.

Reply:
These were all stackviews used/implemented in viewcontroller as bulk. I just decoupled them and did some alterations on them according to new design requirements. They are added and removed according to certain conditions inside the view controller main stackview.